plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/mapping/TableMappingRule.java

```diff
@@ -51,4 +53,23 @@ public String getMapping()
     {
         return mapping;
     }
+
+    @Override
+    public boolean equals(Object o)
```

Comment:
Just `equals` and `hashCode`? I'm not sure how that relates to the commit description ("Concrete the format of JSON identifier mapping") or if it's actually fixing something. Perhaps if I analyzed the tests more closely...

Reply:
It is implemented just to use `isEqualTo` in test.

Reply:
I also do not understand the commit message. Where is the test for which this change is needed?

Reply:
Are we maintaining a cache or meta based on the JSON structure ?

Reply:
No.